### PR TITLE
Cleanup: Logging in Core

### PR DESCRIPTION
**Pull request by chinhodado**
This will also get rid of a few (~15) warnings

### CODE REVIEW

**Comment by bunnei** [comment]
I think you could probably just delete all of that crap... It doesn't do anything but log something that we are not concerned with (that perhaps SkyEye was, at some point)

**Comment by chinhodado** [comment]
Done

**Comment by lioncash** [comment]
Please leave the pr_debug calls that you removed in vfp_double there (or convert them to debug Log calls). They make things significantly less of a headache when debugging the vfp.

File: src/core/arm/skyeye_common/vfp/vfp.cpp [hunk 1]
```diff
@@ -884,13 +775,11 @@ void VMOVR(ARMul_State* state, ARMword single, ARMword d, ARMword m)
...
 int32_t vfp_get_float(arm_core_t* state, unsigned int reg)
 {
-    DEBUG("VFP get float: s%d=[%08x]\n", reg, state->ExtReg[reg]);
     return state->ExtReg[reg];
 }
 
 void vfp_put_float(arm_core_t* state, int32_t val, unsigned int reg)
 {
-    DEBUG("VFP put float: s%d <= [%08x]\n", reg, val);
     state->ExtReg[reg] = val;
```
Comment (lioncash) [inline src/core/arm/skyeye_common/vfp/vfp.cpp]: Leave these in. This is helpful for determining if the correct values are being retrieved from the registers.

File: src/core/arm/skyeye_common/vfp/vfp.cpp [hunk 2]
```diff
@@ -898,13 +787,11 @@ uint64_t vfp_get_double(arm_core_t* state, unsigned int reg)
...
     result = ((uint64_t) state->ExtReg[reg*2+1])<<32 | state->ExtReg[reg*2];
-    DEBUG("VFP get double: s[%d-%d]=[%016llx]\n", reg*2+1, reg*2, result);
     return result;
 }
 
 void vfp_put_double(arm_core_t* state, uint64_t val, unsigned int reg)
 {
-    DEBUG("VFP put double: s[%d-%d] <= [%08x-%08x]\n", reg*2+1, reg*2, (uint32_t) (val>>32), (uint32_t) (val & 0xffffffff));
     state->ExtReg[reg*2] = (uint32_t) (val & 0xffffffff);
     state->ExtReg[reg*2+1] = (uint32_t) (val>>32);
```
Comment (lioncash) [inline src/core/arm/skyeye_common/vfp/vfp.cpp]: Same thing here.

**Comment by chinhodado** [comment]
@lioncash Done

**Comment by bunnei** [comment]
LGTM

**Pull request merged**